package.json

```
@@ -1,41 +1,38 @@
 {
   "name": "leapdao-website",
-  "version": "1.0.0",
-  "main": "index.js",
-  "repository": "git@github.com:parsec-labs/leapdao-website.git",
-  "author": "Alex Lunyov <isuntc@gmail.com>",
-  "license": "MIT",
-  "devDependencies": {
-    "autoprefixer": "^8.0.0",
-    "babel-core": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "cssnano": "^3.10.0",
-    "front-matter": "^2.3.0",
-    "glob": "^7.1.2",
-    "gulp": "^4.0.0",
-    "gulp-babel": "^7.0.1",
-    "gulp-batch": "^1.0.5",
-    "gulp-concat": "^2.6.1",
-    "gulp-livereload": "^4.0.1",
-    "gulp-minisite": "^4.1.2",
-    "gulp-noop": "^1.0.0",
-    "gulp-plumber": "^1.2.0",
-    "gulp-postcss": "^7.0.1",
-    "gulp-rename": "^1.2.2",
-    "gulp-uglify": "^3.0.0",
-    "gulp-watch": "^5.0.0",
-    "gulp-wrap": "^0.14.0",
-    "run-sequence": "^2.2.1",
-    "through2": "^2.0.3",
-    "tinytime": "^0.2.6",
-    "vinyl": "^2.1.0"
-  },
+  "description": "TODO",
+  "version": "0.0.1",
   "scripts": {
-    "test": "echo \"No tests\" && exit 0"
+    "dev": "sapper dev",
```

This will require sapper installed as global module. You can "prefix" script with npx to fix it:
```suggestion
    "dev": "npx sapper dev",
```

---

No, it won't. `sapper` added in package deps so npm will look into `node_modules/.bin` folder first

---

I think, npm scripts use node_modules/.bin/, so no npx or global dep needed

---

My bad, tried to run it before installing, then installed everything and assumed that this was caused by not installing package globally 🙇‍♂️ 
I revoke my proposal.